app/filtering.js

```diff
@@ -468,7 +468,33 @@ module.exports.init = () => {
   })
 }
 
-module.exports.isResourceEnabled = (resourceName) => {
+module.exports.isResourceEnabled = (resourceName, url) => {
```

Comment:
I don't have a good way to do this yet, I'm going to disable it for now in private tabs.